taichi/inc/constants.h

```diff
@@ -10,6 +10,11 @@ constexpr std::size_t taichi_global_tmp_buffer_size = 1024 * 1024;
 constexpr int taichi_max_num_mem_requests = 1024 * 64;
 constexpr std::size_t taichi_page_size = 4096;
 constexpr std::size_t taichi_max_message_length = 2048;
+constexpr std::size_t taichi_result_buffer_entries = 32;
```

Comment:
Does this mean that we can return 32 `uint64`s? But it seems that only the entries `0` and `1` are used?

Reply:
We are using a buffer with `32` entries. Unused buffer entries are not fetched back.